GenerateDocs.ps1

```diff
@@ -165,19 +165,19 @@ function Convert-Syntax($objItem, $hasCmdletBinding) {
 }
 
 function Convert-Parameter($objItem, $commandName) {
-  $parmText = $lineFeed + "###  -$($objItem.name.substring(0,1).ToUpper() + $objItem.name.substring(1))"
+  $paramText = $lineFeed + "###  -$($objItem.name.substring(0,1).ToUpper() + $objItem.name.substring(1))"

```

Comment:
I believe this is a local variable and thus changing it is harmless. I'm also happy to drop it or any other change.